mocked-api/handlers.ts

```diff
@@ -46,17 +48,11 @@ const db = factory({
     transformationTemplate: String,
     action: {
       type: String,
-      parameters: {
-        channel: String,
-        webhookUrl: String,
-      },
+      parameters: String,
```

Comment:
Switching to a string for the parameters property here in order to support any kind of action/source in the mocked APIs.

Parameters are converted to a JSON string when saved, and parsed back to an object when retrieving data.